pkg/registry/core/persistentvolumeclaim/storage/storage.go

```diff
@@ -73,6 +77,43 @@ func (r *REST) ShortNames() []string {
 	return []string{"pvc"}
 }
 
+// defaultOnRead sets interlinked fields that were not previously set on read.
+// We can't do this in the normal defaulting path because that same logic
+// applies on Get, Create, and Update, but we need to distinguish between them.
+//
+// This will be called on both PersistentVolumeClaim and PersistentVolumeClaimList types.
+func (r *REST) defaultOnRead(obj runtime.Object) {
```

Comment:
![if-you-like-it-then-you-should-have-put-a-test-on-it](https://user-images.githubusercontent.com/5595220/124326500-545f0980-db3b-11eb-8794-00eb54cdc6c4.jpg)

Reply:
Good point, I tested the underlying function but this layer deserves a few unit tests. E2E tests will help as well.